src/Tools/ReleaseNotesUtil/RuleInfo.cs

```diff
@@ -53,25 +53,14 @@ public RuleInfo()
 
         // Computed properties.
         public string IdWithHelpLinkMarkdown
-        {
-            get
-            {
-                return !string.IsNullOrWhiteSpace(this.HelpLink)
-                    ? $"[{this.Id}]({this.HelpLink})"
-                    : this.Id ?? string.Empty;
-            }
-        }
+            => !string.IsNullOrWhiteSpace(this.HelpLink)
+                ? $"[{this.Id}]({this.HelpLink})"
+                : this.Id ?? string.Empty;
 
         public string DescriptionOrMessageFormatMarkdown
-        {
-            get
-            {
-                return
-                    (!string.IsNullOrWhiteSpace(this.Description)
-                        ? this.Description
-                        : this.MessageFormat)
-                    ?? string.Empty;
-            }
-        }
+            => (!string.IsNullOrWhiteSpace(this.Description)
+                    ? this.Description
+                    : this.MessageFormat)
+                ?? string.Empty;
```

Comment:
⬆️ This change is the reason I do not like this rule.

Reply:
Is there any ticket on Roslyn to enable some tweak of the rule? We could think of some option that would not suggest expression-body if the content is a ternary for example or if it is a "complex" expression (to be defined).

Reply:
I don't understand how the original code is more readable then the new code though. I think we need to do either of 2 things here:

1. Rewrite the code in block body form breaking down into multiple statements.
2. Use the expression body form suggested in this PR.

I am fine with either approach. I think the form in the original code is probably the least preferred form.

Reply:
Let me know your preferences.

Reply:
We have a value of `when_on_single_line` for this feature.